src/migration_utilities.js

```diff
@@ -9,10 +9,10 @@ const argv = require("yargs")
   .version(false)
   .argv
 
-function isMigrationRequired(network) {
+function isDevelopmentNetwork(network) {
```

Comment:
nit: should we take care of this `fork` replacement of the network string?

Reply:
As we will never have a "-fork" attachement in a development network, it is not necessary. This function is currently always returning the right boolean.